emails/reset-password.pug

```diff
@@ -0,0 +1,7 @@
+extends index
+block title
+    h1 Hi, #{firstName}
+block description
+    p You have requested a password reset for your account at SpaceToStudy. Please click “reset my password” to continue.
+    a(href=`${process.env.CLIENT_URL}`) Reset your password
```

Comment:
where is the correct path and `resetToken`  variable that was supposed to be passed there? contact the guy who is responsible for the reset password page on frontend and discuss it

Reply:
It is going to be changed for the correct path when the page "create new password" is done or at least is in the progress to know the correct route

Reply:
roger